:sparkles: Use notifications-api-common library (#1208)
open-zaak/open-zaak: docs/installation/reference/cli.rst

    After configuring the Notificaties API, send a test nofification to verify the
    setup.

``register_kanalen``

>     Registers notifications channels with the notifications API if it doesn't exist

Reviewer: ```suggestion
    Registers notification channels with the notifications API that don't exist
```
